index.html

```diff
@@ -464,6 +466,8 @@ <h2>Glossary</h2>
 
 <div class="letter_anchor" id="u">U</div>
 
+<p><dfn class="lint-ignore export">Unicameral</dfn>. <a href="https://unicode.org/glossary/#unicameral" target="_blank">Unicode definition</a>: <q>A script that has no case distinctions. Most often used in the context of European alphabets.</q></p>
```

Comment:
This definition is clearly broken too. _Unicameral_ is not "used in the context of European alphabets" except as a distinction _from_ bicameral alphabets.

Reply:
Thanks. I removed that sentence.